tests/lib/rules/jsx-no-leaked-render.js

```diff
@@ -124,6 +124,57 @@ ruleTester.run('jsx-no-leaked-render', rule, {
         }
       `,
     },
+    {
+      options: [{ validStrategies: ['coerce'] }],
+      code: `
```

Comment:
let's have code come before options, and let's be sure to include `output` so the fixer is tested

Reply:
>  let's be sure to include output so the fixer is tested

How? This is a valid case.

Reply:
oh duh, sorry.